contracts/flow/token/BloctoPassStamp.cdc

```diff
@@ -37,37 +62,56 @@ pub contract BloctoPassStamp: NonFungibleToken {
             self.message = message
         }
 
-        pub fun getMessage(): String {
+        access(all)
+        fun getMessage(): String{ 
             return self.message
         }
-    }
 
-    pub resource Collection:
-        NonFungibleToken.Provider,
-        NonFungibleToken.Receiver,
-        NonFungibleToken.CollectionPublic
-    {
+        access(all) view fun getViews(): [Type] {
+            return [
+                Type<MetadataViews.Display>()
```

Comment:
I would recommend including NFTCollectionData and NFTCollectionDisplay here too

Reply:
updated, thanks